specifications/config.md

```diff
@@ -0,0 +1,140 @@
+# Configuration
+
+This document specifies how Stormpath SDKs should handle configuration.
+
+## Processing
+
+The order in which the configuration should be processed.
+
+1. Load configuration according to *loading order*.
+2. Check and apply any *post-processing keys*.
+3. Validate the configuration according to the *validation rules*.
+
+## Loading order
+
+The order in which the configuration should be loaded.
+
+1. Default configuration.
+1. `apiKey.properties` file from `~/.stormpath` directory**.
+2. `stormpath.[json or yaml]` file from `~/.stormpath` directory**.
+3. `apiKey.properties` file from application director**.
+4. `stormpath.[json or yaml]` file from application director**.
+5. Environment variables.
+6. Configuration provided through the SDK client constructor. I.e for the Node.js SDK: `new stormpath.Client({/* config here */});`.
+
+*** Ignore file if empty or doesn't exist.*
```

Comment:
It doesn't start with three stars and end with one, it's '** Ignore file if empty or doesn't exist.' in italic :)